src/feature/featurelistcallback.h

```diff
@@ -41,9 +41,9 @@ bool FeatureCallback_iosOrAndroid() {
 // -------------------------
 
 bool FeatureCallback_annualUpgrade() {
-  if (Constants::inProduction()) {
-    return false;
-  }
+//  if (Constants::inProduction()) {
+//    return false;
+//  }
```

Comment:
Commented instead of removed to more easily revert if needed

Reply:
Let's remove it. It's not much additional effort to revert this patch or rewrite the block later if we need to.